lib/features/theory/screens/selected_theory.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_learn_app/features/theory/screens/theory_screen.dart';
+import 'package:get/get.dart';
+
+import '../models/theory_data_model.dart';
+import 'only_indicator.dart';
+
+class SelectedTheory extends StatefulWidget {
+  const SelectedTheory({super.key});
+
+  @override
+  State<SelectedTheory> createState() => _SelectedTheoryState();
+}
+
+class _SelectedTheoryState extends State<SelectedTheory> {
+  var _selectedIndex = 0;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          InkWell(
+            onTap: () {
+              Get.to(() => TheoryScreen());
+            },
+            child: Container(
+              margin: const EdgeInsets.only(left: 15, top: 50),
+              child: Row(
+                children: [
+                  Icon(Icons.arrow_back),
+                ],
+              ),
+            ),
+          ),
+          SizedBox(
+            height: 100,
+          ),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              ...List.generate(
+                theoryTopic.length,
+                (index) => Indicator(
+                  isActive: _selectedIndex == index ? true : false,
+                ),
+              )
+            ],
+          ),
+          Expanded(
+            child: Container(
+              height: MediaQuery.of(context).size.height,
```

Comment:
take MediaQuery.of(context).size as a variable at the top of method and use .sizeOf()

final screenSize = MediaQuery.sizeOf(context);